config/karma.config.js

```diff
@@ -3,11 +3,12 @@
 
 import webpackConfig from './webpack.config';
 
-var coverageReporters = [{
+const coverageReporters = [{
   type: 'text-summary',
 }];
 
-var newWebpackConfig = Object.assign({}, webpackConfig, {
+// eslint-disable-next-line prefer-const
+let newWebpackConfig = Object.assign({}, webpackConfig, {
```

Comment:
What I'm saying is: if we don't want this to be `const` then we should turn off `prefer-const` instead of marking some things as "ignore prefer-const".

The next value defined is `const reporters` but it gets pushed into in an if just after. This object could probably have the `delete newWebpackConfig.entry` bits removed by adding `entry: undefined` to this construct but I'm not sure how webpack will handle that.